scribus/text/textshaper.cpp

```diff
@@ -312,7 +312,7 @@ static hb_blob_t *referenceTable(hb_face_t*, hb_tag_t tag, void *userData)
 		return NULL;
 	}
 
-	return hb_blob_create((const char *) buffer, length, HB_MEMORY_MODE_WRITABLE, NULL, NULL);
+	return hb_blob_create((const char *) buffer, length, HB_MEMORY_MODE_WRITABLE, buffer, free);
```

Comment:
Hmm, that is how it looked before d80614cb386b676c34a0969580ccb3bf4544cca2, so it doesn’t seem to address the original issue?